client/components/ReportsPage/index.jsx

```diff
@@ -65,6 +70,18 @@ class ReportsPage extends Component {
         });
     }
 
+    setOpenExample(i) {
+        this.setState({
```

Comment:
This isn't a blocker, but the more React-y way to do this is to use the [`updater` function](https://reactjs.org/docs/react-component.html#setstate).

This example would then look something like:

```
this.setState(prevState => {
    return {
        apgExamples: [
               ...prevState.apgExamples.slice(0, i),
                Object.assign(prevState.apgExamples[i], {
                    open: !prevState.apgExamples[i]['open']
                }),
                ...prevState.apgExamples.slice(i + 1) 
        ]
  }
});
```